### PR TITLE
Create bo-nit.json

### PR DESCRIPTION
**Pull request by mrshll1001**
This is a funny one. According to https://impuestos.com.bo/rnd-03-13/ and https://boliviaimpuestos.com/numero-de-identificacion-tributaria-nit/ this is definitely an official list so are likely to be primary. However the list isn't open, only lists of accepted providers are which includes their NIT.

Responding to #402

### CODE REVIEW

File: lists/bo/bo-nit.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": {
+    "en": "Tax Identification Number",
+    "local": "Número de Identificación Tributaria "
+  },
+  "coverage": [
+    "BO"
+  ],
+  "description": {
+    "en": "The NIT or Identify Tax Number is the number assigned by the Tax Administration to a natural person, a legal entity at the time of registration for any economic activity, by virtue of which they are taxable persons of some of the taxes established by Law.\nhttps://impuestos.com.bo/rnd-03-13/"
```
Comment (kd-ods) [inline lists/bo/bo-nit.json:10]: Could be edited a little for clarity. Suggestion:

"The _Número de Identificación Tributaria (NIT)_, or Tax Identification Number, is the number assigned by the National Tax Service of Bolivia to a taxable natural person or legal entity. It is assigned once an individual or entity registers with the Service, acknowledging that their economic activity can, by law, be taxed.\nhttps://impuestos.com.bo/rnd-03-13/"

File: lists/bo/bo-nit.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "structure": [
+    "company/sole_trader"
```
Comment (kd-ods) [inline lists/bo/bo-nit.json:13]: Did you mean, instead, to select just 'company' here? That would make more sense. (I imagine that Trusts and Charities might also be taxable in some situations.)

File: lists/bo/bo-nit.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "code": "BO-NIT",
+  "confirmed": true,
+  "listType": "primary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "guidanceOnLocatingIds": "There are yearly lists of named providers (nomina de proveedores) at the following URL https://www.senapi.gob.bo/contrataciones/nomina-de-proveedores. Open a PDF link and you may find identifiers using the NIT column.",
```
Comment (kd-ods) [inline lists/bo/bo-nit.json:21]: Maybe make this more explicit:

"Although there is no NIT lookup service (whereby the NIT number of an individual or entity can be sought or verified) NITs are published in specific cases. For example, there are yearly lists of named providers to government (nomina de proveedores) at the following URL https://www.senapi.gob.bo/contrataciones/nomina-de-proveedores. Open a PDF link and you may find identifiers using the NIT column."

**Comment by mrshll1001** [comment]
Thanks very much @kd-ods I have made the suggested changes!

**Pull request merged**